internal/db/query.go

```diff
@@ -332,3 +402,32 @@ func (p *PackManager) IsPathPacked(path string) *string {
 
 	return nil
 }
+
+func CreateCache(ctx context.Context, tx pgx.Tx, prefix string) (int64, error) {
+	prefix = filepath.Join(prefix, "%")
+	sql := `
+		WITH impactful_packed_objects AS (
+			SELECT hash, count(*) as count
+			FROM dl.objects
+			WHERE path LIKE $1
+				AND packed = true
+				AND stop_version IS NULL
+			GROUP BY hash
+			ORDER BY count DESC
+			LIMIT 100
```

Comment:
As mentioned in the other PR, should this become a function var and maybe even an argument to the `CreateCache` RPC.